python/tvm/meta_schedule/testing/custom_builder_runner.py

```diff
@@ -167,4 +167,4 @@ def run_module_via_rpc(
         rt_mod = session.load_module(filename)
         dev = session.device(dev_type=dev_type, dev_id=0)
         args = [ndarray.array(arg, dev) for arg in args]
-        return continuation(rt_mod, dev, *args)
+        return continuation(rt_mod, dev, args)
```

Comment:
Note: the intention of the star here is to allow more flexible function signatures like:

```python
def f_timer(mod, dev, a): ...
def f_timer(mod, dev, a, b): ...
def f_timer(mod, dev, a, b, c): ...
def f_timer(mod, dev, a, b, c, d): ...
```

but i guess there is no fundamental difference, so i have no strong opinion here.

Reply:
I did this for models with more than one inputs. It is true that it is not much of a difference. Will try and roll this back

Reply:
This actually won't work if we pass `input_data` as a list of NDArray for models like bert. Should we try a wider range of models in ms, I think we should probably keep this change.